src/controller/jobservice/model.go

```diff
@@ -15,7 +15,6 @@
 package jobservice
 
 import "time"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380